platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java

```diff
@@ -519,8 +527,15 @@ private Subject handleAuthenticationToken(HttpServletRequest httpRequest,
                             if (LOGGER.isTraceEnabled()) {
                                 Element samlToken = ((SecurityAssertion) principal)
                                         .getSecurityToken().getToken();
-                                LOGGER.trace("SAML assertion returned: {}",
-                                        XMLUtils.getStrFromNode(samlToken));
+
+                                TransformerFactory transFactory = TransformerFactory.newInstance();
+                                Transformer transformer = transFactory.newTransformer();
+                                StringWriter buffer = new StringWriter();
+                                transformer
+                                        .setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+                                transformer.transform(new DOMSource(samlToken),
+                                        new StreamResult(buffer));
```

Comment:
I can do that. Looks like this is done in multiple different places, but some places are doing this with different output properties.